lxcfs.c

```diff
@@ -478,7 +478,7 @@ static int lxcfs_readdir(const char *path, void *buf, fuse_fill_dir_t filler, of
 		    filler(buf, "..", NULL, 0) != 0 ||
 		    filler(buf, "proc", NULL, 0) != 0 ||
 		    filler(buf, "cgroup", NULL, 0) != 0)
-			return -EINVAL;
+			return -ENOENT;
```

Comment:
So looking through other fuse examples, everyone else seems to simply ignore filler() errors...  which I don't think is the right thing to do, but ENOENT just seems the wrong value.  Oh actually fill_dir_old() in lib/fuse.c itself returns -ENOMEM if filler fails - that seems best.